pkg/kubectl/resource_printer.go

```diff
@@ -412,6 +462,14 @@ func (h *HumanReadablePrinter) HandledResources() []string {
 	return keys
 }
 
+func (h *HumanReadablePrinter) FinishPrint(output io.Writer) error {
+	if !h.options.NoHeaders && !h.options.ShowAll && h.hiddenPodsNum > 0 {
+		_, err := fmt.Fprintf(output, "\n *Be aware of %d pod(s) is(are) in terminated state and hidden from the displayed list!\n", h.hiddenPodsNum)
```

Comment:
Wherever this print statement ends up going, I think the warning should be printed to Stderr, not output. You can use a private `io.Writer` member on `HumanReadablePrinter` to store `os.Stderr` and change it for testing.